xchange-okcoin/src/main/java/info/bitrich/xchangestream/okcoin/OkCoinV3StreamingExchange.java

```diff
@@ -0,0 +1,62 @@
+package info.bitrich.xchangestream.okcoin;
```

Comment:
Other than the different URL and the use of `OkCoinV3StreamingService` instead of `OkCoinStreamingService`, this class and `OkCoinV3StreamingMarketDataService` don't seem to be very different.  Could we instead introduce an exchange-specific parameter to switch between the two and avoid duplicating lots of code?

See `BitmexStreamingExchange` for an example. You could use an argument called something like `Use_OkCoin_V3_Api`.